Tests/scripts/jira_issues.py

```diff
@@ -25,16 +25,24 @@
 
 
 def generate_ticket_summary(prefix: str) -> str:
-    summary = f"{prefix} failed nightly"
+    summary = f"{prefix} fails nightly"
```

Comment:
```suggestion
    summary = f"{prefix} fails nightly"  # This is the existing conventions of the Content gold bot, don't change as it will break backward compatibility.
```